astroquery/cds/core.py

```diff
@@ -254,7 +254,7 @@ def _args_to_payload(self, **kwargs):
             region = kwargs['region']
             if isinstance(region, MOC):
                 self.path_moc_file = os.path.join(os.getcwd(), 'moc.fits')
-                region.write(format='fits', write_to_file=True, path=self.path_moc_file)
+                region.write(self.path_moc_file, format="fits")
```

Comment:
would this also work with older mocpy? If not, then the minimum version number of 0.5 should be listed in the docs (docs/index.rst).